src/components/AssetLogo.tsx

```diff
@@ -16,54 +16,161 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { Asset } from '@alephium/sdk'
 import { ALPH } from '@alephium/token-list'
-import { Ghost } from 'lucide-react'
-import styled, { css } from 'styled-components'
+import { HelpCircle } from 'lucide-react'
+import { useEffect } from 'react'
+import ReactTooltip from 'react-tooltip'
+import styled, { css, useTheme } from 'styled-components'
 
+import { useAssetMetadata } from '@/api/assets/assetsHooks'
 import AlephiumLogoSVG from '@/images/alephium_logo_monochrome.svg'
 
 interface AssetLogoProps {
-  asset: Pick<Asset, 'id' | 'logoURI' | 'name'>
+  assetId: string
   size: number
   showTooltip?: boolean
   className?: string
 }
 
-const AssetLogo = ({ asset, size, showTooltip, className }: AssetLogoProps) => (
-  <div className={className} data-tip={showTooltip && (asset.name || asset.id)}>
-    {asset.logoURI ? (
-      <LogoImage src={asset.logoURI} />
-    ) : asset.id === ALPH.id ? (
-      <LogoImage src={AlephiumLogoSVG} />
-    ) : (
-      <Ghost size={size * 0.6} />
-    )}
-  </div>
+const AssetLogo = (props: AssetLogoProps) => {
+  const { assetId, showTooltip, className } = props
+
+  const theme = useTheme()
+
+  const metadata = useAssetMetadata(assetId)
+
+  const assetType = metadata.type
+
+  useEffect(() => {
+    metadata && ReactTooltip.rebuild()
+  }, [metadata])
+
+  return (
+    <AssetLogoStyled className={className} {...props}>
```

Comment:
I don't like this `{...props}` solution we thought of anymore. I am convienced that it will create problems. I just had a look at the DOM to see if the `AssetLogoProps` are being passed to the `div` and the `size` is an [actual HTML attribute](https://html.spec.whatwg.org/#the-size-attribute). We can't assume that the dev will know all valid HTML attributes and that they will think of this before passing props, nor do we know what effect some of them might have when the browser renders the DOM. I think we need to find a different solution for accessing React component props in a styled component.

<img width="438" alt="image" src="https://github.com/alephium/explorer/assets/1579899/b7487bee-e4b1-46e2-915a-a3fcbfb54d08">